proto/provenance/name/v1/name.proto

```diff
@@ -40,11 +40,35 @@ message CreateRootNameProposal {
   option (gogoproto.goproto_getters)  = false;
   option (gogoproto.goproto_stringer) = false;
 
-  string title       = 1;
+  // proposal title
+  string title = 1;
+  // proposal description
   string description = 2;
-  string name        = 3;
-  string owner       = 4;
-  bool   restricted  = 5;
+  // the bound name
+  string name = 3;
+  // the address the name will resolve to
+  string owner = 4;
+  // a flag that indicates if an owner signature is required to add sub-names
+  bool restricted = 5;
+}
+
+// ModifyNameProposal details a proposal to update the address
+// and/or the restricted flag of a name.
+message ModifyNameProposal {
```

Comment:
I feel like we should do this as a `gov.v1`-style msg proposal instead of as `gov.v1beta1` content.

I know @llama-del-rey is working on updating our governance proposals to the `gov.v1` style, e.g. #1288. If we just do that right off the bat with this one, we don't end up with as many deprecated proto messages.

Does anyone else have thoughts on this?

Reply:
Sure, I can definitely convert it to that.